CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to OpenGHG will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased](https://github.com/openghg/openghg/compare/0.6.2...HEAD)
+
+## [0.6.2] - 2023-08-xx
+
+### Fixed
+
+- Bug where high time resolution footprints were being clobbered into low
```

Comment:
Maybe something like "Bug when adding footprints where this was unable to distinguish between high time and low time resolution footprints" may be a bit more clear?